django/utils/cache.py

```diff
@@ -411,7 +411,7 @@ def learn_cache_key(request, response, cache_timeout=None, key_prefix=None, cach
     if key_prefix is None:
         key_prefix = settings.CACHE_MIDDLEWARE_KEY_PREFIX
     if cache_timeout is None:
-        cache_timeout = settings.CACHE_MIDDLEWARE_SECONDS
+        cache_timeout = int(settings.CACHE_MIDDLEWARE_SECONDS)
```

Comment:
Can we add tests for these changes?

Reply:
Yes

Reply:
@MiirzaBaig Is there a reason why you leave useless comments?

Reply:
You are right